cli/client.go

```diff
@@ -883,7 +883,25 @@ func (c *viamClient) robotPart(orgStr, locStr, robotStr, partStr string) (*apppb
 			return part, nil
 		}
 	}
-	return nil, errors.Errorf("no machine part found for %q", partStr)
+
+	if robotStr != "" {
```

Comment:
Minor: comments to indicate the 2 cases (machine id vs. machine name) for this if-block and the code above.

Reply:
done